lru_cache/lru_cache.py

```diff
@@ -30,4 +37,40 @@ def get(self, key):
   the newly-specified value. 
   """
   def set(self, key, value):
-    pass
+    if self.queue.len() == 10:
```

Comment:
you've got the limit hard coded here as 10 but it is possible for someone to initialize with a different limit...careful!